### PR TITLE
Docs/getting started contracts

### PR DESCRIPTION
**Pull request by Sc0ra**

### CODE REVIEW

File: user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+Then, we will add this contract library as a dependency for the users-backend and the frontend:
+
+```bash
+cd backend/users
+yarn add @my-project-name/users-contracts
+
+cd ../../frontend
+yarn add @my-project-name/users-contracts
+```
```
Comment (adriencaccia) [inline user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx:60]: Let's move the commands to `pnpm`

File: user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    sources={{
+      light: swarmionAppUsersContractsGraphLight,
+      dark: swarmionAppUsersContractsGraphDark,
+    }}
+  />
+</div>
+
+## Define our first contract
+
+Now lets write our first contract to define a get user operation.
```
Comment (adriencaccia) [inline user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx:86]: We should add a note/section to explain that the contract must be exported by the `src/index.ts` file

File: user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx (hunk 1 above)
Comment (fargito) [reply user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx:60]: Also can we specify to add it in `tsconfig.json` in both consumers?

File: user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+// backend/users/src/functions/getUser/handler.ts
+import { getHandler } from '@swarmion/serverless-contracts';
+import { getUserContract } from '@my-project-name/users-contracts';
+
+import { getUser } from 'libs/getUser';
+
+export const handler = getHandler(getUserContract, async (event) => {
+  const { userId } = event.pathParameters; // userId will be typed as string
+
+  const user = await getUser(userId);
```
Comment (fargito) [inline user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx:174]: Maybe for now just log it and return a fake user for now? As an example

File: user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+import { getUserContract } from '@my-project-name/users-contracts';
+
+export const getUser = (userId: string) => {
+  return getFetchRequest(
+    getUserContract, 
+    fetch,
+    { pathParameters: {
+        userId,
+      },
+    },
```
Comment (fargito) [inline user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx:194]: Isn't there a formatting issue?

**Comment by fargito** [comment]
A few minor changes, but this is awesome @Sc0ra!

File: user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+      light: swarmionAppUsersContractsGraphLight,
+      dark: swarmionAppUsersContractsGraphDark,
+    }}
+  />
+</div>
+
+At last, you should reference the contracts library tsconfig.build.json file in both contract consumers 
+in order to have live type updates without the need to package the contracts library.
+
+```json title="backend/users/tsconfig.build.json"
```
Comment (fargito) [inline user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx:87]:
```suggestion
```json title="backend/users/tsconfig.json"
```

File: user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+    // ...
+    "references": [
+      { "path": "../../contracts/users-contracts" }
+    ]
+  },
+  // ...
+}
+```
+
+```json title="frontend/tsconfig.build.json"
```
Comment (fargito) [inline user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx:99]:
```suggestion
```json title="frontend/tsconfig.json"
```

File: user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+  // ...
+}
+```
+
+## Define our first contract
+
+Now lets write our first contract to define a get user operation.
+
+```typescript
+// contracts/users-contracts/src/contracts/getUser.ts
```
Comment (fargito) [inline user-docs/documentation/docs/getting-started/5-add-a-new-serverless-contract-library.mdx:117]: On peut mettre ça en `title` ?

**Review by fargito — COMMENTED** [review]
Great work!

**Review by fargito — APPROVED** [review]

**Pull request merged**